### PR TITLE
Debug information on nist requests

### PR DESCRIPTION
**Pull request by derIckeBrln**
Added logging on nist requests in order to get and log built urls as well as detailed return codes

### CODE REVIEW

**Review by P-T-I — APPROVED** [review]

**Comment by P-T-I** [comment]
Thanks @derIckeBrln; could you address the black formatting issues?

**Comment by oh2fih** [comment]
Shouldn't these be on the DEBUG logging level instead of INFO, and only be logged when debugging some problem? It seems this would produce a lot of logs especially during initial population.

**Comment by derIckeBrln** [comment]
Good Point, i'll address that and the formatting later this day

**Comment by derIckeBrln** [comment]
Hopefully both fixed
Set normally unneccesary logs to debug-level.
Formatting done, sorry new to python and black

File: CveXplore/core/api_base_class.py [hunk 1]
```diff
@@ -168,7 +167,9 @@ def call(
         """
         Method for requesting free format api resources
         """
-        self.logger.info(f"Making API call with method: {method}, RESSOURCE: {resource}")
+        self.logger.debug(
+            f"Making API call with method: {method}, RESSOURCE: {resource}"
```
Comment (oh2fih) [inline CveXplore/core/api_base_class.py]: Typo:
```suggestion
            f"Making API call with method: {method}, RESOURCE: {resource}"
```

**Comment by derIckeBrln** [comment]
Typo corrected

**Review by oh2fih — APPROVED** [review]

**Review by P-T-I — APPROVED** [review]

**Pull request merged**